pkg/abstract/model/endpoint_rotator_config_test.go

```diff
@@ -174,14 +179,14 @@ func getMonthPartitionedTestHeavy(t *testing.T) {
 			require.Equal(t, monthList[i-(i%partSize)], rc.getMonthPartitioned(month))
 		}
 	}
-
 }
 
 func offsetDateTest(t *testing.T) {
+	t.Setenv("TZ", "Europe/Moscow") // this test is timezone aware
```

Comment:
Moved here to apply to all subtests, and also because setting env *after* calling t.Parallel() might lead to inconsistent results.

Reply:
panic: testing: t.Parallel called after t.Setenv; cannot set environment variables in parallel tests